docs/_docs/configuration/menu-hjson.md

```diff
@@ -362,3 +363,188 @@ newUserApplicationPre: {
     // note that the rest of this menu is omitted for clarity
 }
 ```
+
```

Comment:
Love all your doc work!!!

Reply:
Thanks! As I tell people at work, it doesn't matter if you do something if you don't tell people about it :D I always think updating docs is a big priority, and appreciate all the help you've given on it!